demisto_sdk/commands/validate/validators/BC_validators/BC103_have_the_args_changed.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.tools import compare_lists
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Script
+
+
+class HaveTheArgsChangedValidator(BaseValidator[ContentTypes]):
+    error_code = "BC103"
+    description = "Check if the argument name has been changed."
+    rationale = "If an existing argument has been renamed, it will break backward compatibility."
+    error_message = (
+        "Possible backward compatibility break: Your updates to this file contain changes "
+        "to the names of the following existing arguments: {args}. Please undo the changes."
+    )
+    related_field = "name"  # TODO: check if this is the correct field
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.MODIFIED]
+    related_file_type = [RelatedFileType.YML]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+
+            current_args = [arg.name for arg in content_item.args]
+            old_args = [arg.name for arg in content_item.old_base_content_object.args]  # type: ignore
+            args_diff = compare_lists(sub_list=old_args, main_list=current_args)
+
+            if args_diff:
```

Comment:
Maybe let's also add a check here for cases where we add a new required arg?
This is something that can happen for scripts as well.

Reply:
There is a validation for that IN116- will wait to hear from @JasBeilin